### ISSUE TITLE
Discord Messenger - SECURITY ALERT

### ISSUE THREAD

**Issue by mijjjj4** [DiscordMessenger/dm#125]
WARNING: Your connection may not be private 
When entering a token this message appears, if I click repeat it does not disappear, but if I skip it, I get kicked out of my account

**Comment by mijjjj4** [comment]
I exported and installed the certificate in windows, but still the error appears

**Comment by iProgramMC** [comment]
Which certificate(s) did you install?

If you're confident that you aren't the subject of a MITM attack, then this warning is safe to click "Ignore" on. From then on, all security warnings are ignored and the connection will proceed regardless of perceived certificate validity.

**Comment by mijjjj4** [comment]
I clicked on ignore, but a window pops up that I have to restart the application to customize something or something like that, I restart it and I have to log in there, and then it pops up again.

**Comment by mijjjj4** [comment]
I went to github.com , then clicked connect - certificate and then clicked download, installed and opened the file, then it installed, as reported to me by windows

**Comment by iProgramMC** [comment]
Yes, the app has to restart. Unless it actually crashes for whatever reason after you dismiss the "restarting" dialog it should start working... Check the event viewer for relevant crash logs and send them my way if available :)
